app/models/miq_server/log_management.rb

```diff
@@ -155,7 +156,7 @@ def pg_log_patterns
     [pg_data.join("*.conf"), pg_data.join("pg_log/*")]
   end
 
-  def post_current_logs(taskid)
+  def post_current_logs(taskid, context)
```

Comment:
if we pass a depot, instead of a context, you could actually write tests using a test depot